include/range/v3/view/drop_while.hpp

```diff
@@ -67,6 +67,12 @@ namespace ranges
         }
     };
 
+#if RANGES_CXX_DEDUCTION_GUIDES >= RANGES_CXX_DEDUCTION_GUIDES_17
+    CPP_template(typename Rng, typename Fun)(
+        requires ViewableRange<Rng>)
```

Comment:
`ViewableRange` is again redundant, but we do need to apply some constraints to `Fun` here. We want to stuff it into a `semiregular_t`; `CopyConstructible<Fun>` is the prerequisite.